test/quantization/core/test_workflow_module.py

```diff
@@ -1118,6 +1125,53 @@ def test_fused_mod_reduce_range(self):
         self.assertEqual(obs.quant_min, 0)
         self.assertEqual(obs.quant_max, 127)
 
+    def test_embedding_bag_qat_config(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.emb1 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                self.emb2 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+
+            def forward(self, indices):
+                return torch.cat((self.emb1(indices), self.emb2(indices)))
+
+        model = Model()
+        indices = torch.randint(0, 10, (5, 12))
+        qat_mappings = get_default_qat_module_mappings()
+        float_qparams_fakequant = FakeQuantize.with_args(observer=torch.quantization.PerChannelMinMaxObserver,
+                                                        qscheme=torch.per_channel_affine_float_qparams,
+                                                        ch_axis=0,
+                                                        memoryless=True)
+        float_qparams_qconfig = QConfig(activation=NoopObserver,
+                                        weight=float_qparams_fakequant)
+        model.qconfig = float_qparams_qconfig
+
+        quant_model = torch.quantization.QuantWrapper(model)
+        quant_model = torch.quantization.prepare_qat(quant_model, mapping=qat_mappings)
+
+        count_fake_quant = 0
+        for name, mod in quant_model.named_modules():
+            if name.endswith('weight_fake_quant'):
+                count_fake_quant += 1
+                self.assertEqual(type(mod), FakeQuantize)
+
+        self.assertEqual(count_fake_quant, 2)
+
+        quant_model(indices)
+
+        convert_mappings = get_default_static_quant_module_mappings()
+        # For EmbeddingBags, input should not be quantized.
+        convert_mappings[torch.ao.quantization.stubs.QuantStub] = torch.nn.quantizable.Identity
```

Comment:
@supriyar As discussed offline, input to embeddings should not be quantized.  A follow-up task should be making this more automatic for eager mode QAT, otherwise this would preclude quantizing other modules.